internal/model/state.go

```diff
@@ -265,11 +290,13 @@ func newRollupsStateInspect(
 	}
 }
 
-func (s *rollupsStateInspect) finish(status cModel.CompletionStatus) {
+func (s *rollupsStateInspect) finish(status cModel.CompletionStatus) error {
 	s.input.Status = status
-	s.input.ProcessedInputCount = s.getProcessedInputCount()
+	inputCount, _ := s.getProcessedInputCount()
```

Comment:
In this part, we need to verify the error.

Reply:
done